Makefile

```diff
@@ -123,26 +123,27 @@ cluster-api-provider-azure: $(CONTROLLER_GEN)
 api-docs: $(GENAPIDOCS)
 	hack/gen-api-docs.sh $(GENAPIDOCS) $(DIR)
 
+# generate the SelectorSyncSet template file to be applied via app-interface
 .PHONY: app-sre-saas-template
 app-sre-saas-template: hypershift
+	# do not set oidc-storage-provider-s3-* parameters as these are not used
 	bin/hypershift install \
-		--oidc-storage-provider-s3-bucket-name=bucket \
-		--oidc-storage-provider-s3-secret=oidc-s3-creds \
-		--oidc-storage-provider-s3-region=us-east-1 \
-		--oidc-storage-provider-s3-secret-key=credentials \
```

Comment:
@csrwng can you confirm we can now just remove those settings if we pre-provision OIDC documents in the S3 bucket ?

Reply:
Yes you can just remove them.